### PR TITLE
Added framework for 7/(how many) kits

### PR DESCRIPTION
**Pull request by yambotzion**

### CODE REVIEW

File: src/com/znd/smash/commands/SmashChamps.java [hunk 1]
```diff
@@ -56,4 +56,124 @@ private static ItemStack createPyroMenuKit() {
 		
 		return item;
 	}
+
```
Comment (desmondcchi) [inline src/com/znd/smash/commands/SmashChamps.java:68]: You need to incorporate these item icons within the createChampMenu() function - above.

File: src/com/znd/smash/commands/SmashChamps.java [hunk 2]
```diff
@@ -56,4 +56,124 @@ private static ItemStack createPyroMenuKit() {
...
+		lore.add(ChatColor.RED + "Click to select kit!");
+		meta.setLore(lore);
+		
+		meta.addEnchant(Enchantment.DURABILITY, 1, true);
+		meta.addItemFlags(ItemFlag.HIDE_ENCHANTS);
+		item.setItemMeta(meta);
+		
+		return item;
+	}
+
```
Comment (desmondcchi) [inline src/com/znd/smash/commands/SmashChamps.java:187]: Please add a screenshot of the GUI in the PR. Look up how to set up a spigot server and export the plugin as a jar then add it in the plugins folder.

**Review by desmondcchi — CHANGES_REQUESTED** [review]

File: src/com/znd/smash/commands/SmashChamps.java (hunk 2 above)
Comment (yambotzion) [reply src/com/znd/smash/commands/SmashChamps.java:187]: Screenshot 2022-09-11 230637

File: src/com/znd/smash/commands/SmashChamps.java (hunk 1 above)
Comment (yambotzion) [reply src/com/znd/smash/commands/SmashChamps.java:68]: done

Comment (desmondcchi) [reply src/com/znd/smash/commands/SmashChamps.java:68]: I don't see the updated code.

**Review by desmondcchi — CHANGES_REQUESTED** [review]

File: src/com/znd/smash/commands/SmashChamps.java (hunk 2 above)
Comment (desmondcchi) [reply src/com/znd/smash/commands/SmashChamps.java:187]: Good job!

File: src/com/znd/smash/commands/SmashChamps.java (hunk 1 above)
Comment (desmondcchi) [reply src/com/znd/smash/commands/SmashChamps.java:68]: Nevermind, I see it. My bad.

**Review by desmondcchi — APPROVED** [review]
LGTM

**Pull request merged**